src/Audio_details.md

```diff
@@ -0,0 +1,192 @@
+# Audio Details
+
+tl;dr:
+> The PPU is a bunch of state machines, and the APU is a bunch of counters.
+
+<!-- Styles in <svg> tags apply to the whole page, and we have two on this one. Hoist this to be less confusing. -->
+<style>
+  text {
+    fill: var(--fg);
+    dominant-baseline: middle;
+    font-size: .6em;
+  }
+  #dsg { font-size: 1em; }
+  .centered { text-anchor: middle; }
+  .right    { text-anchor: end; }
+  .tabnum { font-feature-settings: "tnum"; }
+  svg > rect, svg > use {
+    stroke: var(--fg);
+    fill: var(--fg);
+  }
+  svg > path, svg > circle {
+    stroke: var(--fg);
+    fill: none;
+  }
+  .dashed { stroke-dasharray: 2; }
+  .opa50 { opacity: .5; }
+  .fill50 { fill-opacity: .5; }
+  .digital   { stroke: #df0000; fill: #df0000; }
+  .analog    { stroke: #005f98; fill: #005f98; }
+  .misc_data { stroke: #a4269c; fill: #a4269c; }
+  .dac_ena   { stroke: #ffa700; fill: #ffa700; }
+  .ch_active { stroke: #17cc00; fill: #17cc00; }
+  rect.legend { width: 5px; height: 5px; }
+  text.legend { font-size: .7em; }
+  .arrow { stroke-width: 2px; opacity: .35; }
+  .unfilled {
+    fill: none;
+  }
+  .no-stroke {
+    stroke: none;
+  }
+</style>
+
+<figure>
+{{#include imgs/apu_detailed.svg:2:}}
+
+<figcaption>Source: Lior "LIJI32" Halphon</figcaption>
+</figure>
+
+Each of the four "conceptual" channels is composed of a "generation" circuit (designated "channel" in the above diagram), and a [DAC](https://en.wikipedia.org/wiki/Digital-to-analog_converter).
+The digital value produced by the generator, which ranges between $0 and $F (0 and 15), is linearly translated by the DAC into an analog[^digital_analog] value between -1 and 1 (the unit is arbitrary).
+
+The four analog channel outputs are then fed into the mixer[^vin], which selectively adds them (depending on [`NR51`](<#FF25 — NR51: Sound panning>)) into two analog outputs (Left and Right).
+Thus, the analog range of those outputs is 4× that of each channel, -4 to 4.
+
+Then, both of these two get their respective volume scaled, once from [`NR50`](<#FF24 — NR50: Master volume & VIN panning>), and once from the volume knob (if the console has one).
+Note that the former step never mutes a non-silent input, but the latter can.
+
+Each of the two analog outputs then goes through a [high-pass filter](https://en.wikipedia.org/wiki/High-pass_filter) (HPF).
+For short, a HPF constantly tries to "pull" the signal towards analog 0 (neutral); the reason for that is explained further below.
+
+[^digital_analog]:
+To be clear: digital values are discrete and clear-cut; conversely, the analog domain is continuous.
```

Comment:
I feel like this clarification shouldn't be here. Maybe it's something we can add to an initial glossary/list of necessary concepts needed to read Pan Docs?

Reply:
Sure, but this PR has waited long enough that I don't want it to become blocked by #194 (which itself has been twiddling its thumbs for a long while now).

Reply:
Added to that issue